packages/kbn-std/src/pick.ts

```diff
@@ -7,11 +7,12 @@
  */
 
 export function pick<T extends object, K extends keyof T>(obj: T, keys: readonly K[]): Pick<T, K> {
-  return keys.reduce((acc, key) => {
-    if (obj.hasOwnProperty(key)) {
```

Comment:
in the case of http/2, `headers` object is created via `Object.create(null)` so it hasn't got `hasOwnProperty` method